Code/Tools/FBuild/FBuildCore/BFF/BFFParser.cpp

```diff
@@ -790,7 +790,12 @@ bool BFFParser::ParseIncludeDirective( BFFIterator & iter )
     AStackString<> includeToUse;
     if (PathUtils::IsFullPath(include) == false)
     {
-        AStackString<> fileName( iter.GetFileName() );
+        AStackString<> fileName;
+        const char * iterName = iter.GetFileName();
+        if ( iterName )
+        {
+            fileName += iterName;
+        }
```

Comment:
a BFFIterator should never have a nullptr for a filename. In what situation did you see this?

Reply:
Not sure where I saw this. Doesn't seem to be a problem now. Disregard?

Reply:
Works for me :)